src/superphot_plus/ztf_transient_fit.py

```diff
@@ -294,216 +204,10 @@ def create_logL(cube):
     eq_wt_samples = dyfunc.resample_equal(samples, weights, rstate=rstate)
 
     if plot:
-        plt.errorbar(
-            tdata[bdata == "g"],
-            fdata[bdata == "g"],
-            yerr=ferrdata[bdata == "g"],
-            c="g",
-            label="g",
-            fmt="o",
-        )
-        plt.errorbar(
-            tdata[bdata == "r"],
-            fdata[bdata == "r"],
-            yerr=ferrdata[bdata == "r"],
-            c="r",
-            label="r",
-            fmt="o",
-        )
-
-        trange_fine = np.linspace(np.amin(tdata), np.amax(tdata), num=500)
-
-        for sample in eq_wt_samples[:30]:
-            plt.plot(
-                trange_fine,
-                flux_model(sample, trange_fine, ["g"] * len(trange_fine)),
-                c="g",
-                lw=1,
-                alpha=0.1,
-            )
-            plt.plot(
-                trange_fine,
-                flux_model(sample, trange_fine, ["r"] * len(trange_fine)),
-                c="r",
-                lw=1,
-                alpha=0.1,
-            )
-
-        plt.xlabel("MJD")
-        plt.ylabel("Flux")
-        plt.title(prefix)
-        if t0_lim is None:
-            plt.savefig(os.path.join(FIT_PLOTS_FOLDER, prefix + ".png"))
-        else:
-            plt.savefig(os.path.join(FIT_PLOTS_FOLDER, prefix + "_%.02f.png" % t0))
-        plt.close()
-
+        plot_sampling_lc_fit(prefix, FIT_PLOTS_FOLDER, tdata, fdata, ferrdata, bdata, eq_wt_samples)
     return eq_wt_samples
 
 
-def run_curve_fit(filename):
```

Comment:
This entire looks like it is providing a different workflow. Have you confirmed with Kaylee that this isn't being called manually from some notebook somewhere?

Reply:
I'll just leave it in for now.